spec/models/dish_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Dish, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
A workflow comment: At this point you should have written specs for the Dish model.